app/controllers/games_controller.rb

```diff
@@ -3,6 +3,12 @@ def new
     @game = Game.new
   end
 
+  def check
+    @game = current_game
+    @game.check?
+    flash[:notice] = 'You are in Check'
+  end
+
```

Comment:
For this, is the plan to be calling this endpoint from the frontend with ajax?

An alternative, would be to add `flash[:notice] = 'You are in Check' if @game.check?` in the update action.